src/plone/restapi/serializer/utils.py

```diff
@@ -23,13 +23,15 @@ def resolve_uid(path):
     if match is None:
         return path, None
 
-    uid, suffix = match.groups()
+    uid, suffix, anchor = match.groups()
     brain = uuidToCatalogBrain(uid)
     if brain is None:
         return path, None
     href = brain.getURL()
     if suffix:
         return href + "/" + suffix, brain
```

Comment:
What if there is a suffix and an anchor?

Reply:
Done